crates/libs/registry/src/key.rs

```diff
@@ -20,7 +20,7 @@ impl Key {
         let result = unsafe {
             RegCreateKeyExW(
                 self.0,
-                pcwstr(path).as_ptr(),
+                encode_utf16(path).as_ptr(),
```

Comment:
I just renamed `pcwstr` to `encode_utf16` to distinguish between UTF-16 encoding and the "wide" encoding provided by `OsString`.